eclair-core/src/test/scala/fr/acinq/eclair/channel/states/a/WaitForAcceptChannelStateSpec.scala

```diff
@@ -147,6 +159,14 @@ class WaitForAcceptChannelStateSpec extends TestkitBaseClass with StateTestsHelp
     awaitCond(alice.stateName == CLOSED)
   }
 
+  test("recv AcceptChannel (wumbo size channel)", Tag("wumbo"), Tag("min-depth"), Tag("max-funding-size"))  { f =>
```

Comment:
The `min-depth` tag isn't used anymore, and tags should describe what it does (example: the tag "mainnet" isn't called "chainhash")

```suggestion
  test("recv AcceptChannel (wumbo size channel)", Tag("wumbo"), Tag("high-max-funding-size"))  { f =>
```

Reply:
I squashed and rebased the commits to include this nit as well in the fixup.